runtime/src/main/java/org/corfudb/runtime/collections/Table.java

```diff
@@ -322,6 +324,62 @@ public void preCommitCallback(TokenResponse tokenResponse) {
         return keyOfQueueEntry;
     }
 
+    /**
+     * Apply a Corfu SMREntry directly to a stream. This can be used for replaying the mutations
```

Comment:
This can be reworded as - 'Appends the specified element to the end of this unbounded queue, without materializing the queue in memory'

Reply:
Reworded the description!